src/Administration/Resources/app/administration/src/module/sw-category/page/sw-category-detail/index.js

```diff
@@ -636,7 +636,7 @@ Component.register('sw-category-detail', {
         },
 
         saveSlotConfig() {
-            if (this.category.cmsPage.locked || Object.values(this.category.slotConfig).length < 1) {
+            if ((this.category.cmsPage && this.category.cmsPage.locked) || Object.values(this.category.slotConfig).length < 1) {
```

Comment:
[optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) could be used here :slightly_smiling_face: